pytorch_lightning/trainer/trainer.py

```diff
@@ -292,6 +294,26 @@ def __init__(
                 .. deprecated:: 0.5.0
                     Use `min_nb_epochs` instead. Will remove 0.8.0.
 
+            max_steps (int): Stop training after this number of steps. Disabled by default (None).
+                Training will stop if max_steps or max_epochs have reached (earliest).
+                Example::
```

Comment:
I do not think this needs an example, rather write two sentences, what is the default value...
And you already have the description fine, so just drop the example :]